### PR TITLE
docs(react-native): UI component - LobbyView

### PR DESCRIPTION
**Pull request by khushal87**

### CODE REVIEW

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx [hunk 1]
```diff
@@ -3,7 +3,60 @@ id: lobby-view
...
+
+  const onCallJoinHandler = useCallback(async () => {
+    try {
+      // Handle what should happen before joining the call. Eg: showing loader, etc.
+      await call?.join({ create: true });
+      // Handle what should happen after the call is joined. Eg: navigation, etc.
+    } catch (error) {
+      // Handle the case when there is an error while joining the call
+    }
+  }, [call, appStoreSetState]);
```
[the remark this answers is not in the document]
Comment (khushal87) [reply packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx:37]: By mistake, let me remove it

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx [hunk 2]
```diff
@@ -3,7 +3,60 @@ id: lobby-view
 title: LobbyView
 ---
 
-:::info
-This guide is still in progress and will be updated soon.
-If you have any questions, please reach out to us on our [website](https://getstream.io/video/#contact).
-:::
+LobbyView is a component that is designed to make your entrance and invite experience to the meeting user-friendly. This mostly contains all the necessary information about the meeting including:
+
```
Comment (santhoshvai) [inline packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx:7]:
```suggestion
LobbyView is a component that is designed to make your entrance and invite experience to a meeting call user-friendly. The purpose is to allow users to test their setup before entering the meeting call. This component mainly contains all the necessary information about the meeting, including:

```

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx [hunk 3]
```diff
@@ -3,7 +3,60 @@ id: lobby-view
...
+- The call details such as its id, etc.
+- Participants who are already in the meeting.
+- Control the pre-join audio/video mute status.
+- Handle the join button which is used to join the meeting.
```
Comment (santhoshvai) [inline packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx:11]:
```suggestion
- The call details such as its id, etc.
- Participants who are already in the meeting.
- Controlling the audio/video mute status before joining the meeting.
- Exposes a handler for the join meeting button.
```

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx [hunk 4]
```diff
@@ -3,7 +3,60 @@ id: lobby-view
...
+
+## General usage
+
+```tsx
+import { useCallback } from 'react';
+import {
+  Call,
+  LobbyView,
+  StreamCall,
+  useCall,
```
Comment (santhoshvai) [inline packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/lobby-view.mdx:23]:
```suggestion
  LobbyView,
  useCall,
```

**Comment by santhoshvai** [comment]
@khushal87 the screenshot attached doesnt the number of active participants.. is it possible to change the screenshot to one that has it please

**Comment by khushal87** [comment]
@santhoshvai I have added the screenshot. Thanks 😄

**Pull request merged**